materialize-dynamodb/.snapshots/TestValidate

```diff
@@ -539,4 +539,45 @@
 }
 --- End root document projection change - delta updates ---
 
+--- Begin [field: key] unsatisfiable type change with backfill counter increment ---
+{
+	"flow_document": {
+		"type": 2,
+		"reason": "The root document is required for a standard updates materialization"
+	},
+	"flow_published_at": {
+		"type": 4,
+		"reason": "This field is able to be materialized"
+	},
+	"key": {
```

Comment:
For these snapshots, the main difference is from the `unsatisfiable type change` test is that the `key` field has an unsatisfiable type change, but when incrementing the backfill counter the constraints are as if it is a new binding, rather than a change from the original.